app/presenters/booking_presenter.rb

```diff
@@ -0,0 +1,15 @@
+class BookingPresenter < SimpleDelegator
+
+  def self.paypal_url(root_path, id, cost, passenger)
```

Comment:
Unused method argument - `root_path`. If it's necessary, use `_` or `_root_path` as an argument name to indicate that it won't be used.<br>Unused method argument - `id`. If it's necessary, use `_` or `_id` as an argument name to indicate that it won't be used.